packages/mobile/src/fiatExchanges/CashInSuccess.tsx

```diff
@@ -0,0 +1,116 @@
+import Button from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { StackScreenProps } from '@react-navigation/stack'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { cicoProviderSupportEmails } from 'src/config'
+import { Namespaces } from 'src/i18n'
+import { fiatExchange } from 'src/images/Images'
+import { noHeaderGestureDisabled } from 'src/navigator/Headers'
+import { navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { navigateToURI } from 'src/utils/linking'
+
+type RouteProps = StackScreenProps<StackParamList, Screens.CashInSuccess>
+type Props = RouteProps
+
+function CashInSuccessScreen({ route }: Props) {
+  const { t } = useTranslation(Namespaces.fiatExchangeFlow)
+
+  const { provider } = route.params
+  const providerSupportEmail = provider ? cicoProviderSupportEmails[provider] : null
+
+  const openProviderSupportEmail = () => {
+    if (providerSupportEmail) {
+      // TODO: Add non-native email client support by generalizing the mail function
```

Comment:
Looks like the function in `SupportContact` is just a couple lines of code, would you mind adding the generalization in this PR? I don't see a good reason to postpone it :)

If you run into any trouble while doing this or want to discuss the approach feel free to ping me on Slack and I can give you a hand.

Reply:
I wanted to ask Cal whether or not this is actually useful before implementing it. I'm not sure if there is value in making it easy to email provider support immediately after initiating a transaction. I think that sets the wrong user expectation that the transaction should be instant and if it's not then you should email support.

Reply:
That makes sense